package.json

```diff
@@ -9,8 +9,8 @@
     "lint": "eslint 'client/**/*.js' && sass-lint -v",
     "postinstall": "npm run build",
     "starterapp": "webpack-dev-server --config webpack/starterApp.js --devtool eval --progress --colors --hot --history-api-fallback",
-    "test": "env NODE_PATH=$NODE_PATH:$PWD/client ./node_modules/.bin/mocha --opts client/__tests__/mocha.opts",
-    "test:debug": "./node_modules/.bin/karma start client/__tests__/karma.conf.js",
+    "test": "env NODE_PATH=$NODE_PATH:$PWD/client ./node_modules/.bin/mocha-webpack",
+    "test:debug": "webpack-dev-server --config webpack/test.js --hot --inline",
```

Comment:
It was unclear to me which webpack config was being called from test and test:debug, because line 12 doesn't point directly to a config file. Also, **test:debug ** uses **test**.js, while **test** uses **mocha**.js.